js-miniapp-sample/src/pages/landing.js

```diff
@@ -101,6 +103,16 @@ const Landing = (props: LandingProps) => {
     }
   }, [props]);
 
+  useEffect(() => {
+    try {
+      if (document.readyState === 'complete') {
+        miniAppDidFinishLoad();
+      }
+    } catch (e) {
+      console.log(e);
+    }
+  }, []);
+
```

Comment:
Try moving this to App.js

Reply:
Done moved this to App.js and now showing the toast on success of this api call.